crates/wit-component/tests/components/lift-options/module.wit

```diff
@@ -33,6 +33,6 @@ interface my-default {
   p: func() -> list<v-no-string>
 }
 
-world my-world {
-  default export my-default
+default world my-world {
+  export foo: self.my-default
```

Comment:
Agreed! Initially when I talked with Luke about this he thought it would be best to leave out the splat syntax for now, but given that this PR doesn't yet fully implement types-in-worlds there's no true replacement for `default export` just yet.

I'm curious, do you think that releasing these changes and/or merging this PR should be gated on figuring this out? I could go either way myself.